<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PrivateChat</class>
 <widget class="QWidget" name="PrivateChat">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>678</width>
    <height>544</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>私聊ing...</string>
  </property>
  <property name="windowIcon">
   <iconset resource="imgs.qrc">
    <normaloff>:/img/kita2.jpg</normaloff>:/img/kita2.jpg</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">/* 主窗口 */
QMainWindow{
    background-color: #f5f5f5;  /* QQ风格的浅灰色背景 */
    font-family: &quot;Microsoft YaHei&quot;, &quot;Segoe UI&quot;, sans-serif;
    font-size: 12px;
}

QToolButton{
	background-color: rgba(0, 0, 0,0);
	border: none;
}

QToolButton#btnClose:hover{
	background-color: rgb(255, 0, 0);
}

/* 应用标题 */
#labelTitle {
    color: purple;
    font-size: 16px;
    font-weight: bold;
    padding-left: 5px;
}

QTextEdit#Edit_communicate{
	background-color: #f5f5f5;
    border: 1px solid #dcdcdc;
	border-radius: 5px;
    padding: 5px;
    selection-background-color: #e0f0ff;
    selection-color: black;
}

/* 聊天输入框 */
QTextEdit#say_textEdit {
    background-color: #f5f5f5;
    font-size: 13px;
	border:none;
}

/* 发送按钮 */
#btnSay {
    background-color: #0084ff;  /* QQ蓝色 */
    color: white;
    border: none;
    border-radius: 4px;
    font-size: 13px;
    font-weight: 500;
    min-width: 80px;
    min-height: 32px;
}

#btnSay:hover {
    background-color: #0073e6;
}

#btnSay:pressed {
    background-color: #005bbf;
}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="labelTitle">
       <property name="text">
        <string>Ave Mujica</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QToolButton" name="btnClose">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="imgs.qrc">
         <normaloff>:/img/close.png</normaloff>:/img/close.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="spacing">
      <number>5</number>
     </property>
     <item>
      <widget class="QWidget" name="widget_bottom" native="true">
       <layout class="QGridLayout" name="gridLayout_3">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item row="3" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="btnSay">
            <property name="text">
             <string>发送</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="2" column="0">
         <widget class="QTextEdit" name="say_textEdit">
          <property name="minimumSize">
           <size>
            <width>600</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>700</width>
            <height>100</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QTextEdit" name="Edit_communicate">
          <property name="minimumSize">
           <size>
            <width>600</width>
            <height>350</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>700</width>
            <height>16777215</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="imgs.qrc"/>
 </resources>
 <connections/>
</ui>
